site/guides/porch-ui-installation.md

```diff
@@ -0,0 +1,50 @@
+# Accessing the Configuration as Data UI
+
+The easiest way to access the Configuration as Data UI is running by a docker
+container on your local machine where you'll be able to access the UI with your
+browser. Running the container locally simplifies the overall setup by allowing
+the UI to use your local kubeconfig and Google credentials to access the GKE
+cluster with Porch installed. This guide will show you how to do this.
+
+For Production, the Configuration as Data UI can be installed by adding the
```

Comment:
Added some subsections would be useful.

## Installing in Kubernetes
...

## Running locally in a container
...

Reply:
Added `Running in Backstage` subsection since at this point, the plugin can be configured to be ran locally or within Kubernetes. I added this section after 'Running locally in a container' since adding the plugin to Backstage is more time consuming process.